<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KumppaniTuoteWidget</class>
 <widget class="QWidget" name="KumppaniTuoteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="view">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="tuoNappi">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Tuo</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/tuotiedosto.png</normaloff>:/pic/tuotiedosto.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="VieNappi">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Vie</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/vie.png</normaloff>:/pic/vie.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="laskutaNappi">
       <property name="text">
        <string>Laskuta</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/lasku.png</normaloff>:/pic/lasku.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="raporttiButton">
       <property name="text">
        <string>Tulosta</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/print.png</normaloff>:/pic/print.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="uusiNappi">
       <property name="text">
        <string>Uusi</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/lisaa.png</normaloff>:/pic/lisaa.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="muokkaaNappi">
       <property name="text">
        <string>Muokkaa</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/muokkaa.png</normaloff>:/pic/muokkaa.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yhdistaButton">
       <property name="text">
        <string>Yhdist√§</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/kopioi.png</normaloff>:/pic/kopioi.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="poistaNappi">
       <property name="text">
        <string>Poista</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/roskis.png</normaloff>:/pic/roskis.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
